tests/test_root_versioning.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+
+"""
+<Program Name>
+  test_root_versioning.py
```

Comment:
I guess we can append `_integration` to these types of tests, including `test_key_revocation.py`

Examples:
test_keydb.py (unit test)
test_replay_attack.py (attack test)
test_root_versioning_integration.py (integration test)
test_key_revocation_integration.py (integration test)